### DISCUSSION TITLE
Applications keep getting denied

### DISCUSSION THREAD

**Discussion by Tau-be** [community/community#54527] [GitHub Education]
### Hi everyone,

I'm have sent over 12 applications already, provided all the proof I can acquire yet still am being denied permanently. My classmates could verify after sending only a single document please help me out here this is incredibly frustrating.

Edit: Has been resolved after spamming the same Student ID

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
